packages/server/modules/core/domain/projects/logic.ts

```diff
@@ -0,0 +1,39 @@
+const adjectives = [
```

Comment:
Isn't this just a helper? Why put it in the `domain` folder? + The name is too generic (logic.ts)